### ISSUE TITLE
Issue: Nanominer problem

### ISSUE THREAD

**Issue by Byrd1083** [RainbowMiner/RainbowMiner#513]
![image](https://user-images.githubusercontent.com/42227809/56776454-61c12200-6781-11e9-81d0-b7482935b183.png)

**Comment by hernandn01** [comment]
Also review GPU selection, Nanominer is trying to mine on all cards, while I have Ethash disable for 1060 3GB and it is clearly being use by Trex at the moment
![image](https://user-images.githubusercontent.com/41829368/56782204-63034680-67a3-11e9-824f-4e497e30cc35.png)

**Comment by RainbowMiner** [comment]
Oh dear! I will check this miner again.

**Comment by RainbowMiner** [comment]
Could you eventually send me the Debug file from this rig? Also, is this a mixed GPU rig? Like Intel+Nvidia or Nvidia+Amd? 
If you find the time, please run GPUtest.bat and send me the result file.
Nanominer needs a different GPU selection, compared to all the other miners.

**Comment by RainbowMiner** [comment]
Ok, nanominer had a bug in v1.2.2 where the GPU device selection failed. It has been fixed in new version v1.2.3.

**Comment by Byrd1083** [comment]
Okay, I tried to rebench again. Seems to throw an error about no wallet or payment ID set. Yet, it is benching against miningpoohub. I have checked and they do not have this type of setup to mine to a wallet so I am not sure what to put there as it should not be needed with that pool. Still needs fixing ;-)
![image](https://user-images.githubusercontent.com/42227809/56819954-a4294400-67ff-11e9-9a58-03341e56234f.png)
